waffle_hub/dataset/adapter/autocare_dlt.py

```diff
@@ -94,6 +94,8 @@ def export_autocare_dlt(self, export_dir: Union[str, Path]) -> str:
         _export_autocare_dlt(self, export_dir, train_ids, val_ids, test_ids, [])
     elif self.task == TaskType.INSTANCE_SEGMENTATION:
         _export_autocare_dlt(self, export_dir, train_ids, val_ids, test_ids, [])
+    elif self.task == TaskType.SEMANTIC_SEGMENTATION:
```

Comment:
NIT: If it's a code that checks if it's a valid task, I think we can reduce the number of codes.